2-mandatory/3-financial-times.js

```diff
@@ -5,16 +5,25 @@
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
 function potentialHeadlines(allArticleTitles) {
-    // TODO
+       if(allArticleTitles.lengh !== 0){
+        allArticleTitles = ARTICLE_TITLES.filter((el) => el.length <= 65);
```

Comment:
here you are using a mixture of allArticleTitles which is passed into the function and ARTICLE_TITLES (just use allArticlesTitles). Good use of the filter array method, but you don't need the if statement, for example if you do `let shortTitles = allArticleTitles.filter((el) => el.length <= 65)` and then return this, it will return an empty array if allArticleTitles.length is 0 :)